script/inline_schemas/inline_schemas.go

```diff
@@ -12,11 +14,11 @@ const basePath = "./docs/spec/"
 
 func main() {
 	schemaPaths := []struct {
-		path, schemaOut, goVariable, packageName string
+		path, schemaOut, goVariable string
 	}{
-		{"errors/payload.json", "processor/error/schema.go", "errorSchema", "error"},
-		{"transactions/payload.json", "processor/transaction/schema.go", "transactionSchema", "transaction"},
-		{"sourcemaps/payload.json", "processor/sourcemap/schema.go", "sourcemapSchema", "sourcemap"},
+		{"errors/payload.json", "processor/error/generated-schemas/payload.go", "PayloadSchema"},
+		{"transactions/payload.json", "processor/transaction/generated-schemas/payload.go", "PayloadSchema"},
+		{"sourcemaps/payload.json", "processor/sourcemap/generated-schemas/payload.go", "SourcemapSchema"},
```

Comment:
The name `SourcemapSchema` is not consistent - please also change to `PayloadSchema`.